kedro/framework/session/session.py

```diff
@@ -49,21 +49,25 @@ def _deactivate_session() -> None:
 
 def _describe_git(project_path: Path) -> Dict[str, Dict[str, Any]]:
     project_path = str(project_path)
-
     try:
         res = subprocess.check_output(
-            ["git", "rev-parse", "--short", "HEAD"], cwd=project_path
+            ["git", "rev-parse", "--short", "HEAD"],
+            cwd=project_path,
+            stderr=subprocess.STDOUT,
+        )
+        git_data = {"commit_sha": res.decode().strip()}  # type: Dict[str, Any]
+        res = subprocess.check_output(
```

Comment:
Minor: but I would rename this perhaps to `git_status_res` just to avoid confusion with the variable above.
```suggestion
        git_status_res = subprocess.check_output(
```